google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -12,25 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
-
 #include "google/cloud/pubsub/internal/tracing_message_batch.h"
 #include "google/cloud/pubsub/internal/message_batch.h"
 #include "google/cloud/pubsub/version.h"
 #include "google/cloud/internal/opentelemetry.h"
+#include <algorithm>
+#include <memory>
+#include <string>
```

Comment:
It doesn't look like these are needed outside of `GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY`, so why move them?